src/App.js

```diff
@@ -5,25 +5,27 @@ import UserOutput from './UserOuput/UserOutput';
 
 class App extends React.Component {
   state = {
-    username: 'Olyaaaa',
+    text: 'Olyaaaa',
   }
 
-  userInputHandler = (event) => {
-    if (event.key === "Enter") {
-      this.setState({
-        username: event.target.value,
-      })
-    }
+  handleTextInput = (event) => {
+    this.setState({
+      text: event.target.value,
+    })
   }
 
-  render() {
-    const size = 2;
-    const outputElements = Array(size).fill( <UserOutput username={this.state.username}/>);
+  handleLetterDeletion = (index) => {
+    const text = [...this.state.text];
```

Comment:
Pretty hacky way :)

I also write code like that,  but `this.state.text.split('')` is more common